Milestone 1: ERD

SofiBretz opened the pull request:
### Building Facebook Milestone 1: ERD

In this milestone we:

- Created the Entity Relationship Diagram (ERD).
- Thought about friendship and friendship requests, also, the Post related to the comments and likes. 
- Created the associations: User - Post, User - Friendship, Post - Comments and Post - Like.
- Added the .png on the docs on the project, on the readme file.

samgaco commented:
Hi @SofiBretz :high_brightness: 

Before anything:

- [ ] Are you the only one working in this project?

Then, some issues with the milestone:

- [ ] The ERD should be in a docs folder. It's said in the requirements in pathwright. ([Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/final_project/milestone1/erd2.png))

- [ ] Since a user can give many likes, a user will have many likes, this relation can't be seen in your ERD.

- [ ] A comment should also be related to a user, not only to the post.

Please address those mentioned above, and submit back for another review. Good spirit!! :muscle: 

-----------------------

* ###### STATUS OF THE PROJECT: REQUESTED CHANGES :stop_sign:

SofiBretz commented:
### Building Facebook Milestone 1: ERD

In this milestone we:

- Updated the ERD file.
- Added a folder named 'docs' and a file with ERD file on .pdf.
- Readme image updated.
